torchao/prototype/mx_formats/mx_tensor.py

```diff
@@ -60,12 +61,27 @@
     unpack_uint4,
 )
 
+SBITS, EBITS_F32, MBITS_F32 = 1, 8, 23
+EBITS_F4_E2M1, MBITS_F4_E2M1 = 2, 1
+EBITS_F4_E3M0, MBITS_F4_E3M0 = 3, 0
+EBITS_F6_E2M3, MBITS_F6_E2M3 = 2, 3
+EBITS_F6_E3M2, MBITS_F6_E3M2 = 3, 2
+EBITS_F8_E4M3, MBITS_F8_E4M3 = 4, 3
+EBITS_F8_E5M2, MBITS_F8_E5M2 = 5, 2
+
+
+class ScaleCalculationMode(Enum):
```

Comment:
thanks! If there is any data you can share on these options (as comments around this enum), that would be awesome.